CMakeLists.txt

```diff
@@ -55,6 +55,10 @@ IF(MSVC)
     add_compile_options("$<$<CXX_COMPILER_ID:MSVC>:/utf-8>")
 ENDIF(MSVC)
 
+IF(APPLE)
+	add_compile_definitions("TURN_NO_THREAD_BARRIERS=1")
+endif()
```

Comment:
consistent casing plz "IF/ENDIF"

Reply:
Right, I had reservations about that
I see different casing in different CMakeList files.